torch/fx/experimental/sym_node.py

```diff
@@ -281,121 +285,121 @@ def _graph_repr(self) -> builtins.str:
 
     # These methods call the metaprogrammed methods, they're hand written
     # here so we get good stack traces
-    def abs(self) -> "SymNode":
+    def abs(self) -> SymNode:
```

Comment:
oops, I regret adding annotations to this file lol

Reply:
@ezyang, just do typing_extensions Self instead?